projects/packages/status/src/class-errors.php

```diff
@@ -2,6 +2,7 @@
 /**
  * An errors utility class for Jetpack.
  *
+ * @deprecated since $$next-version$$
```

Comment:
I meant the comment below on lines 16–18 🙂

Reply:
Ah 😊 Changed in https://github.com/Automattic/jetpack/pull/37451/commits/859b7bf7f3d07ba055ace0f07df8fb8272df8ae4